db/migrate/20150709221755_create_logins.rb

```diff
@@ -1,4 +1,4 @@
-class CreateLogins < ActiveRecord::Migration
+class CreateLogins < ActiveRecord::Migration[4.2]
```

Comment:
Do you like Clearance's idea of injecting the version if the Rails version is >= 5.0?